includes/fields/class-field-url.php

```diff
@@ -26,15 +26,16 @@ public function render( $field_settings, $form_id ) {
 
             <div class="wpuf-fields">
                 <input
-                    id="<?php echo esc_attr( $field_settings['name'] ) . '_' . esc_attr( $form_id ); ?>"
-                    type="url" class="url <?php echo ' wpuf_'. esc_attr( $field_settings['name'] ).'_'. esc_attr( $form_id ); ?>"
-                    data-duplicate="<?php echo esc_attr( $field_settings['duplicate'] ) ? esc_attr( $field_settings['duplicate'] ) : 'no'; ?>"
-                    data-required="<?php echo esc_attr( $field_settings['required'] ) ?>"
-                    data-type="text"
-                    name="<?php echo esc_attr( $field_settings['name'] ); ?>"
-                    placeholder="<?php echo esc_attr( $field_settings['placeholder'] ); ?>"
-                    value="<?php echo esc_attr( $value ); ?>" size="<?php echo esc_attr( $field_settings['size'] ); ?>"
-                    autocomplete="url"
+                    title = "The URL must contain a protocol i.e. http://example.com"

```

Comment:
You'll want to use [esc_attr__](https://developer.wordpress.org/reference/functions/esc_attr__/) to print that title.

Take a look at [this page](https://pippinsplugins.com/localizing-and-translating-wordpress-plugins/) and focus on the **Step 2 – Localize Your Plugin Text** section.